api/email_messages.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "lorawan-stack/api/client.proto";
+import "lorawan-stack/api/rights.proto";
+
+package ttn.lorawan.v3;
+
+option go_package = "go.thethings.network/lorawan-stack/v3/pkg/ttnpb";
+
+// CreateClientEmailMessage is used as a wrapper for handling the email
+// regarding the create client procedure.
```

Comment:
nit; no need for Go-style comments

Reply:
It does propagate to the created struct, which is a bit useful to reinforce that it serves as a wrapper while on development ( mouse over variable definition on vscode ).